.azure/templates/jobs/build_java.yaml

```diff
@@ -24,6 +24,11 @@ jobs:
       - template: '../steps/prerequisites/install_java.yaml'
         parameters:
           JDK_VERSION: $(jdk_version)
+      # Build the Java code without tests
```

Comment:
As far as I can see this isn't being invoked yet, is the expectation that it will be needed when we add the system tests?

Reply:
I think that is because the PR is from fork.

Reply:
It is needed because of the spotbugs plugin, which needs the modules to be build before it runs the checks -> as you will get the `dependency issue found` type of error otherwise.

Reply:
The exact error is shown here: https://dev.azure.com/cncf/strimzi/_build/results?buildId=171513&view=logs&j=2de397de-21fc-5e8b-587d-4865bafb3e06&t=d9c95e40-2ff1-577f-4ff1-10fe2890ceac

```
[ERROR] Failed to execute goal on project operator: Could not resolve dependencies for project io.strimzi.kafka.access:operator:jar:0.0.1-SNAPSHOT: Could not find artifact io.strimzi.kafka.access:api:jar:0.0.1-SNAPSHOT -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project operator: Could not resolve dependencies for project io.strimzi.kafka.access:operator:jar:0.0.1-SNAPSHOT: Could not find artifact io.strimzi.kafka.access:api:jar:0.0.1-SNAPSHOT

```

Reply:
Ok, thanks